.github/workflows/tests.yml

```diff
@@ -108,7 +177,9 @@ jobs:
           cp ../.github/sqlite.testing.env testing.env
 
       - name: Test
-        run: go test ./... -v -database -filesystem -p 1 -coverprofile=coverage.txt -covermode=atomic -json > test-report.json
+        run: |
+          go test ./... -v -database -filesystem -p 1
+          go test ./... -v -database -filesystem -p 1 -coverprofile=coverage.txt -covermode=atomic -json > test-report.json
```

Comment:
@googollee, BTW, I have a question about running unit tests for Golang:
In this repo, I have SonarCloud integrated, which needs the unit test execution report in JSON format, but after I added the corresponding parameter to the run tests command, its output switched totally to JSON and the default plain test output is unavailable. Is there a way to run the tests only once and get both: plain text default output to STDOUT and JSON report to a file?
Now to have the ability to analyse test runs in the GitHub Actions UI, I have to run tests twice(

Reply:
@googollee, just checking whether you missed this question from the discussion thread or just didn't have time to look into it yet?)

Reply:
Oh, I missed this. I don't think there is a way to output different formats at the same time. I accept the test log as JSON format.